pubspec.yaml

```diff
@@ -4,9 +4,6 @@ version: 0.15.1
 author: Luan Nico <luannico27@gmail.com>
 homepage: https://github.com/flame-engine/flame
 
-flutter:
```

Comment:
????

Reply:
lol, I removed in the wrong place, it is the heat....

Reply:
I no longer have that problem with the heat ;)